aiida_wannier90/io/_group_list.py

```diff
@@ -19,10 +19,9 @@ def group_list(values):
         else:
             groups.append(sorted(set([current_start, v1])))
             current_start = v2
-    # final group
-    else:
+        # final group
         groups.append(sorted(set([current_start, v2])))
```

Comment:
Here the ``groups.append`` should happen only after the for-loop has finished -- that is, both ``groups.append`` and ``return groups`` should be indented one more level to the left. Sorry, my previous comment wasn't clear on this point.

Reply:
Yes, sorry!